src/test/java/uk/gov/pay/api/filter/RateLimiterFilterTest.java

```diff
@@ -44,13 +47,13 @@ public void setup() {
 
         Account account = new Account(ACCOUNT_ID, TokenPaymentType.CARD, "some-token-link");
         SecurityContext mockSecurityContext = mock(SecurityContext.class);
-        when(mockSecurityContext.getUserPrincipal()).thenReturn(account);
+        lenient().when(mockSecurityContext.getUserPrincipal()).thenReturn(account);
```

Comment:
I’d prefer it we moved these so they’re only in methods where they’re actually used.

Reply:
yeah ok I've updated 👍